src/integrations/Braze/browser.js

```diff
@@ -35,62 +36,16 @@ class Braze {
     }
 
     this.name = NAME;
-    this.previousPayload = null;
     this.supportDedup = config.supportDedup || false;
-    this.areTransformationsConnected = destinationInfo && destinationInfo.areTransformationsConnected;
-    this.destinationId = destinationInfo && destinationInfo.destinationId;
+    const { areTransformationsConnected, destinationId } = destinationInfo || {};
+    this.areTransformationsConnected = areTransformationsConnected;
+    this.destinationId = destinationId;
```

Comment:
Nice.
You can further simplify it.

`({areTransformationsConnected: this.areTransformationsConnected, destinationId: this.destinationId} = destinationInfo ?? {});`

Another option is to declare these variables in the class beforehand and simply do this:
`
({areTransformationsConnected, destinationId} = destinationInfo ?? {});
`

Note: I've used the null coalescing operator, as it is more appropriate here.

Reply:
This has been addressed